### PR TITLE
SAV-55: add pics and meta info to blog posts

### PR DESCRIPTION
**Pull request by AnneTee**
@kostajh I think this is ready. Let me know if you have any more suggestions! :cookie:

### CODE REVIEW

**Comment by kostajh** [comment]
I like it! @dmurphy1, @timstallmann and @chrisarusso, what do y'all think?

**Comment by dmurphy1** [comment]
@kostajh, @AnneTee - I think it looks good. Not much of a concern to me, but it does look like the 3 recent posts are by the same author (and maybe they will be once we have more up).

**Comment by AnneTee** [comment]
@dmurphy1 Yeah, I think that's a product of not having much content. We actually might make those blog posts "related posts" based on tags once we have more content, but for now it's just the 3 most recent posts that aren't the post you're looking at.

**Comment by chrisarusso** [comment]
I like it. One thing I noticed @AnneTee is that it seems the author's name is hardcoded to savaslabs.com instead of relative... it brought me to the live site when perusing locally.

**Comment by kostajh** [comment]
@timstallmann want to take a look before this gets merged?

**Comment by AnneTee** [comment]
@chrisarusso good call, I updated the code to replace the absolute links with relative :tangerine:

**Comment by timstallmann** [comment]
Looks good to me, thanks @AnneTee !

File: _layouts/event.html [hunk 1]
```diff
@@ -9,8 +9,9 @@
 
   <header class="post-header">
     <h2 class="post-title">{{ page.title }}</h2>
-    <p class="event-meta">{{ page.date | date: "%b %-d, %Y" }}{% if page.author %} • <a href="{{ page.author | downcase | replace: ' ', '-'  | prepend: "/team/" | prepend: site.baseurl | prepend: site.url | strip_newlines }}">{{ page.author }}</a>{% endif %}{% if page.meta %} • {{ page.meta }}{% endif %}</p>
+    <p>{{ page.date | date: "%b %-d, %Y" }} • {{ tags_content }}</p>
```
Comment (kostajh) [inline _layouts/event.html]: @AnneTee could you please replace this with `&centerdot;` (http://dev.w3.org/html5/html-author/charref for reference)

File: _layouts/post.html [hunk 2]
```diff
@@ -8,14 +8,15 @@
 
   <header class="post-header">
     <h2 class="post-title">{{ page.title }}</h2>
-    <p class="post-meta">{{ page.date | date: "%b %-d, %Y" }}{% if page.author %} • <a href="{{ page.author | downcase | replace: ' ', '-'  | prepend: "/team/" | prepend: site.baseurl | prepend: site.url | strip_newlines }}">{{ page.author }}</a>{% endif %}{% if page.meta %} • {{ page.meta }}{% endif %}</p>
+    <p>{{ page.date | date: "%b %-d, %Y" }} • {{ tags_content }}</p>
```
Comment (kostajh) [inline _layouts/post.html]: Same as above

**Comment by AnneTee** [comment]
@kostajh done, although I had to do some tweaking since the centerdot character is tiny compared to whatever we were using before :diamond_shape_with_a_dot_inside:

**Comment by kostajh** [comment]
thanks @AnneTee! Open to changing centerdot. `&circledast;`?

**Comment by AnneTee** [comment]
@kostajh not really, but I'd be fine with using &tacocat;

**Pull request merged**